packages/jest-emotion/package.json

```diff
@@ -23,12 +23,12 @@
     "@emotion/core": "^10.0.10",
     "dtslint": "^0.3.0",
     "emotion": "^10.0.9",
-    "enzyme-to-json": "^3.3.5",
+    "enzyme-to-json": "^3.2.1",
```

Comment:
why the downgrades? 🤔

Reply:
It was done automatically because the root depended on `^3.2.1`. Since nothing is breaking, I'm not going to change it.